config/routes.rb

```diff
@@ -149,7 +149,9 @@
     get '/revision_history/:id/revert_object' => 'versions#revert_object', as: 'revision_history_revert_object'
     get '/revision_history/:id/revert_attribute' => 'versions#revert_attribute', as: 'revision_history_revert_attribute'
   end
-  resources :organizations, only: [:index]
+  resources :organizations, only: [:index] do
+    get :conferences
```

Comment:
How about 
```
member do
  get :conferences
done
```
so that CanCanCan works?

Reply:
@differentreality updated in https://github.com/openSUSE/osem/pull/1757/commits/09c43b1e34a471a48f561bb50e6ba474ef0942a3